.github/workflows/conformance.yml

```
@@ -0,0 +1,292 @@
+name: Talos Conformance
+on:
+  pull_request_target:
+    types:
+      - opened
+      - synchronize
+      - reopened
+  schedule:
+    # Run weekly.
+    - cron: '* * * * 1'
+  push:
+    branches:
+      - main
+  pull_request:
+jobs:
+  setup-and-test:
+    runs-on: ubuntu-22.04
+    permissions:
+      id-token: write
+      contents: read
+    strategy:
+      fail-fast: false
+      max-parallel: 4
+      matrix:
+        include:
+          # --- Cilium v1.15 ---
+
+          - name: 'v1.15'
+            cilium-version: "v1.15.1"
+            talos-version: 'v1.6.2'
+            kube-proxy: false
+            kube-proxy-replacement: "true"
+            socketlb: false 
+            bpf-masquerade: true
+            ipam-mode: 'kubernetes'
+            ipv4: true
+            ipv6: false
+            encryption-enabled: false
+            encryption-type: ipsec
+            tunnel-mode: vxlan
+            nodeport: true
+
+          - name: 'v1.15 w wireguard'
+            cilium-version: "v1.15.1"
+            talos-version: 'v1.6.2'
+            kube-proxy: true 
+            kube-proxy-replacement: "false"
+            socketlb: false 
+            bpf-masquerade: false
+            ipam-mode: 'kubernetes'
+            ipv4: true
+            ipv6: false
+            encryption-enabled: true
+            encryption-type: wireguard
+            tunnel-mode: vxlan
+            nodeport: false
```

```suggestion
            nodeport: true
```

Would be required for `bpf-masquerade: true`, no? https://docs.cilium.io/en/stable/network/concepts/masquerading/#ebpf-based